### ISSUE TITLE
画像の表示について

### ISSUE THREAD

**Issue by tommytome** [NIFTYCloud-mbaas/UserCommunity#167]
初心者です。
現在ファイルストアへアップした画像をhtmlページに表示したく考えております。
http://blog.mb.cloud.nifty.com/?p=328
上記ページを参考しましたが、アップ後にページにアクセスしても、上手くされません。

<img src="ファイル名.jpg" >
通常のwebサイトと同じく上記のような記載だけでは表示されないのでしょうか。

※下記のページを見て、公開ファイル設定などもしてみましたが上手くいっておりません。
http://mb.cloud.nifty.com/doc/dashboard/filestore.html

**Comment by moongift** [comment]
@tommytome 

どういったURLでアクセスしているでしょうか？後、ファイルのパーミッションを全員に公開にしていますでしょうか？

**Comment by tommytome** [comment]
早速のご回答ありがとうございます。
パーミッションは全員に公開しています。

URLは、「<img src="アプリケーションID/publicFiles/ファイル名.jpg" >」で書いてみています。

**Comment by moongift** [comment]
@tommytome 

アプリケーションIDの前に/は入っているでしょうか？後は日本語ファイル名だともしかするとうまく表示されないかも知れませんがいかがでしょう？

**Comment by tommytome** [comment]
はい、アプリケーションIDの前に/は入っています。また日本語ファイル名ではない形式でやっておりました、、が、画像自体が表示されず、画像マーク？のようなものがでる状態です。

htmlファイル以外に、どこかのjsファイル等の設定が何か必要でしょうか、、？？

**Comment by moongift** [comment]
@tommytome 

一つ忘れていました。アプリ設定にある、**HTTPSでの取得** は有効になっているでしょうか？

**Comment by tommytome** [comment]
はい、
アプリ設定＞データ・ファイルストア＞公開ファイル設定　　で
HTTPSでの取得は有効になっています。

**Comment by moongift** [comment]
@tommytome 
そうですねぇ…例えば

https://mb.api.cloud.nifty.com/2013-09-01/applications/xVk8i5MZQZLNGVmi/publicFiles/randing.html

のように表示できるのですが。URLの形は同じようになっていますでしょうか？アプリケーションIDはランダムな文字列なので、適切にコピーされていますでしょうか？

```
https://mb.api.cloud.nifty.com/2013-09-01/applications/xVk8i5MZQZLNGVmi/publicFiles/randing.html
```

**Comment by tommytome** [comment]
はい、、上記のURLを参考にすると、画像を表示させる場合のパスは/applications以降のURLを記載し、最後にranding.htmlではくてその画像ファイルのファイル名を記載する格好ですよね、、？
そうなってはいますが、、;

**Comment by moongift** [comment]
@tommytome 
URLに 2013-09-01 は入っていますか？後は実際、どういった画面が表示されているのでしょう？エラーの状態が分からないのでなんとも…。

**Comment by tommytome** [comment]
![4](https://cloud.githubusercontent.com/assets/5723755/7786775/adf8bc0e-021e-11e5-9807-9edc4477d96a.png)

**Comment by moongift** [comment]
@tommytome 
ドメインが入っていませんね。今の状態だと、アプリのローカルを見ています。

**Comment by tommytome** [comment]
ドメインが入ってない、、というのは下記の画像のように頭をつけるということでしょうか
![5](https://cloud.githubusercontent.com/assets/5723755/7786976/44b25e22-022c-11e5-8499-5036ac4b1dd3.png)

**Comment by moongift** [comment]
@tommytome 
https:// も必要です。Monacaを使っていますので、 /applications/ 〜だけだとMonacaアプリ以下を指し示しています。

**Comment by tommytome** [comment]
https://mb.api.cloud.nifty.com/2013-09-01　までを頭に入れて、表示させることができました！
大変感謝致します。

**Comment by moongift** [comment]
:+1:
